gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/IcebergMetadataWriter.java

```diff
@@ -1081,6 +1092,9 @@ public void writeEnvelope(RecordEnvelope<GenericRecord> recordEnvelope, Map<Stri
     }
   }
 
+  public void sendAuditCounts(String topicName, List<String> serializedAuditCountMaps) {
```

Comment:
Add javadoc to this method explaining the intention. Also, can we change to Collection<String> instead of List<String>